docs/README-EDITING.md

```diff
@@ -0,0 +1,174 @@
+| `permalink`    | The page's **unique** URL.                                 | Yes      |
+| `canonicalUrl` | A canonical URL of the page's latest version.              | Yes      |
+| `metaTitle`    | The page's SEO meta title.                                 | No       |
+| `tags`         | Tags used by the documentation search engine.              | No       |
+
+Frontmatter example:
+
+```
+---
+title: Introduction
+permalink: /next/api/
+canonicalUrl: /api/
+---
+```
+
+### Sitemap frontmatter
+
+The [`vuepress-plugin-sitemap`](https://www.npmjs.com/package/vuepress-plugin-sitemap) lets us use additional tags to customize the documentation site map:
+
+```
+---
+sitemap:
+  exclude: false
+  changefreq: hourly
+---
+```
+
+## Editing the documentation
+
+When editing the documentation content, follow the guidelines below.
+
+### Editing the `next` documentation version
+
+Draft files to be included in the documentation's next version are kept in the `next` directory.
+
+The `next` version of the documentation is available only locally and on the staging server.
+
+To display the `next` version in a browser:
+1. Start a [local Handsontable documentation server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/next/.
+
+### Editing a published documentation version
+
+To edit an already-published documentation version, go to the required [`<semver.version>` directory](./README.md#handsontable-docs-directory-structure) (e.g. `9.0`).
+
+To display a published documentation version in a browser:
+1. Start a [local Handsontable documentation server](./README.md#getting-started-with-handsontable-docs).
+2. In your browser, go to http://localhost:8080/docs/{semver.version}/.
+
+If you're editing the `latest` version (a version with the largest `<semver.version>` number), remember to make the same edits to the `next` version as well.
+
+## Editing the API reference
+
+The API reference is generated automatically from the source code, into the `/next/api/` directory.
+
+To edit the API reference, go into the source code and change the JSDoc comments.
+
+To generate the API reference output, run the following command from the `/docs/` directory:
+
+```bash
+npm run docs:api
+```
+
+### JSDoc links
+
+As we want to support case-insensitive URLs, follow these guidelines when editing the JSDoc links:
+
+* Use lower case characters.
+* To separate words before `#`, use `-`.
+
+```js
+// Wrong:
+/** {@link Options#autoColumnSize}. */
+
+// OK: 
+/** {@link options#autocolumnsize Options#autoColumnSize}. */
+
+
+// Wrong:
+/** {@link HiddenRows} */
+
+// OK:
+/** {@link hidden-rows HiddenRows} */
+```
+
+## Documentation versioning
+
+To create a new version of the Handsontable documentation:
+
+* From the `handsontable/docs` directory, run:
+```bash
+npm run docs:version <semver.version>
+# for example:
+# npm run docs:version 9.0
```

Comment:
Even if I put the version number in the command's argument like shown above, I have to answer the question about the version to generate :) Is it expected behaviour?

![image](https://user-images.githubusercontent.com/11849621/123823717-be8a6b00-d8fd-11eb-9c5b-d861d35d694c.png)

Reply:
In #8255 the README was updated:

https://github.com/handsontable/handsontable/pull/8255/files#diff-d7e656edc6b6824b076b84acdec3ba38907deeac521cd5e691f26b1914773d41L130-L134

Reply:
And a day later, #8248 overwrote changes from #8255.

Reply:
@swistach @Siemienik I updated README-EDITING.md: https://github.com/handsontable/handsontable/pull/8348